dev-util/tup/tup-0.7.10.ebuild

```diff
@@ -0,0 +1,103 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit linux-info toolchain-funcs
+
+DESCRIPTION="A file-based build system"
+HOMEPAGE="http://gittup.org/tup"
+# Tup itself is GPLv2, but it bundles differently licensed software:
+# - lua: MIT
+# - sqlite (unused in this ebuild): public domain
+# - inih: 3-clause BSD
+# - red-black tree implementation: 2-clause BSD
+# - queue implementation: 3-clause BSD
+LICENSE="GPL-2 MIT public-domain BSD BSD-2"
+SLOT="0"
+
+if [[ "${PV}" == 9999 ]]; then
+	EGIT_REPO_URI="https://github.com/gittup/tup.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/gittup/tup/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+DEPEND="
+	dev-db/sqlite:=
+	dev-libs/libpcre:=
+	sys-fs/fuse:=
+"
+RDEPEND="${DEPEND}"
+
+CONFIG_CHECK="~FUSE_FS ~NAMESPACES"
+WARNING_FUSE_FS="CONFIG_FUSE_FS is required for tup to work"
+WARNING_NAMESPACES="CONFIG_NAMESPACES is required for tup to work as intended (workaround: set TUP_NO_NAMESPACING env var when running tup)"
+
+src_prepare() {
+	# Use our toolchain
+	sed -i Tuprules.tup \
+		-e "s:CC = gcc:CC = $(tc-getCC) ${CFLAGS} ${LDFLAGS}:" \
+		-e "s:ar crs:$(tc-getAR) crs:"
```

Comment:
You shouldn't use `:` as a delimiter when modifying `CFLAGS` with `sed`, 
https://bugs.gentoo.org/685160.

Also add `|| die` at the end of this command, 
https://devmanual.gentoo.org/ebuild-writing/error-handling/index.html

Reply:
Done